src/reconcile/rec_write.c

```diff
@@ -210,6 +210,14 @@ __reconcile(WT_SESSION_IMPL *session, WT_REF *ref, WT_SALVAGE_COOKIE *salvage, u
         WT_STAT_CONN_INCR(session, cache_write_restore);
         WT_STAT_DATA_INCR(session, cache_write_restore);
     }
+    if (!WT_IS_HS(btree)) {
```

Comment:
Would it be useful to have another set of stats for HS? I think the one with prepare might be, I have not followed on with the prepare project so need to ask someone else, @kommiharibabu  maybe?

Reply:
I didn't understand any reason why these stats are only considered for tables other than HS? None of the updates in the history store will have `prepare` flag set, but there can be many pages in HS with ts/txn.

Reply:
@kommiharibabu - For now, our end goal with this ticket is to track performance/file size regressions in **data store** associated with storing post durable history metadata(ts, txn ids, prepare support).

@sulabhM, I hope my explanation is correct. We can create another ticket to track the stats for history store as well.

Reply:
Cool, let's include stats for HS as a separate ticket.